ipatests/test_integration/test_advise.py

```diff
@@ -141,7 +141,7 @@ def test_advice_enable_admins_sudo(self):
 
     def test_advice_config_server_for_smart_card_auth(self):
         advice_id = 'config_server_for_smart_card_auth'
-        advice_regex = r"\#\!\/bin\/sh.*"
+        advice_regex = r"\#\!\/bin\/bash.*"
```

Comment:
Why do you change this?
`/bin/sh` is the expected value, while as I see, the actual one comes from https://github.com/freeipa/freeipa/blob/master/ipaserver/advise/base.py#L495 :
```
    def print_header(self, header, print_shell=False):
        header_size = len(header)

        prefix = ''
        if print_shell:
            prefix = '# '
            print('#!/bin/sh')
```
and it also should be `/bin/sh`.

Reply:
ha, this might've been a mistake in the first place, unlike the other one which fixed actual tests

Reply:
dropped